flatpak/io.github.qtox.qTox.json

```diff
@@ -0,0 +1,157 @@
+{
+    "app-id": "io.github.qtox.qTox",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.10",
```

Comment:
I think we should update to 5.11 here, in order to fix https://github.com/flathub/io.github.qtox.qTox/issues/3